master/reference/calicoctl/resources/workloadendpoint.md

```diff
@@ -60,24 +61,24 @@ spec:
 
 #### Metadata
 
-| Field       | Description                 | Accepted Values   | Schema | Default    |
-|-------------|-----------------------------|-------------------|--------|------------|
-| name      | The name of this workload endpoint resource. Reqired. |  Alphanumeric string with optional `.`, `_`, `-`, or `/` | string | |
-| namespace | Namespace provides an additional qualification to a resource name. | | map | "default" |
+| Field     | Description                                                        | Accepted Values                                          | Schema | Default   |
+|-----------|--------------------------------------------------------------------|----------------------------------------------------------|--------|-----------|
+| name      | The name of this workload endpoint resource. Required.             |  Alphanumeric string with optional `.`, `_`, `-`, or `/` | string |           |
+| namespace | Namespace provides an additional qualification to a resource name. |                                                          | map    | "default" |
```

Comment:
I know you didn't change in this PR, but namespace isn't a `map` - it's a string just like the name

Reply:
OK hm, that's not good, will correct this.